doc/licenses/additional.rst

```diff
@@ -43,7 +43,7 @@ from implementations in Geant4::
       software itself.
 
    3. The names "Geant4" and "The Geant4 toolkit" may not be used to endorse or
-      promote software,or products derived therefrom, except with prior written
+      promote software,or products derived there from, except with prior written
```

Comment:
Humm ... Did we copy/paste this from G4? Since it is there license, should we changing it?